### DISCUSSION TITLE
Unable to run Cube demo on iOS simulator

### DISCUSSION THREAD

**Discussion by paradescence** [KhronosGroup/MoltenVK#2130] [Q&A] [answered]
When I open the Cube demo project in xcode, it builds and the app appears in the iOS simulator home screen, but the app immediately exits when it is launched with the following error message:

"Unable to find the Vulkan runtime on the system.
This likely indicates that no Vulkan capable drivers are installed."

I've installed and built everything following the instructions in the README.md, and while the MacOS build works for the Cube demo and runs correctly, I'm unable to get the iOS build to run.

**Comment by billhollings** [comment]
Thanks for submitting this. I've created issue #2134 to track this, and I am working on a fix.

**Comment by billhollings** [comment, marked as answer]
PR #2136 fixes this, and is now available in the latest MoltenVK.

**Reply by paradescence** [reply]
Thank you! I've just downloaded the updated code, rebuilt moltenvk, and confirmed that the cube demo is now working correctly for me in the ios simulator when I build and run it. Thanks again!
